xls/modules/fp_fast_rsqrt_32.x

```diff
@@ -0,0 +1,109 @@
+// Copyright 2021 The XLS Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file implements floating point fast (approximate) 
```

Comment:
Can you add a word or two on how this differs from a...non-fast? non-approximate? sqrt? Just enough to know when I should use this impl. instead of another that may exist at some point.

Reply:
Done.

Reply:
Done.